adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/FromFileRawEntityProvider.java

```diff
@@ -47,16 +52,31 @@ public FromFileRawEntityProvider(RawFileInfo file) throws IOException {
         headerCount = headerCountArray[0];
     }
 
+    /**
+     * Returns a boolean indicating if a GTFS CSV file has more rows that can be read. 1 if yes, else 0.
```

Comment:
Nit (a small detail) - `1` should be `true` in the comments and `0` should be `false`, as `true` and `false` are the boolean literal values.

Reply:
Indeed